src/main/java/org/jboss/xavier/sanitizer/XavierPayloadSanitizer.java

```diff
@@ -0,0 +1,169 @@
+package org.jboss.xavier.sanitizer;
+
+import com.jayway.jsonpath.*;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+
+public class XavierPayloadSanitizer
+{
+    private static String inputFileName;
+    private static String outputFileName;
+    private static String issuesConditionsJSONFileName;
+
+    public static void main(final String[] args)
+    {
+        List<String> argsList = Arrays.asList(args);
+        argsList.forEach(arg -> {
+            if( arg.equals("--input"))
+            {
+                inputFileName = argsList.get(argsList.indexOf(arg) + 1);
+            }
+            if( arg.equals("--output"))
+            {
+                outputFileName = argsList.get(argsList.indexOf(arg) + 1);
+            }
+
+            if( arg.equals("--issues"))
+            {
+                issuesConditionsJSONFileName = argsList.get(argsList.indexOf(arg) + 1);
+            }
+        });
+
+        String jsonFileToBeSanitized = readInputFile(inputFileName, "Input file to sanitize");
+
+        String jsonIssuesConditions = readInputFile(issuesConditionsJSONFileName, "Issues Conditions file ");
+        List<String> vmsIssues = splitConditions(jsonIssuesConditions,"###vms");
+        List<String> hostsIssues = splitConditions(jsonIssuesConditions,"###hosts");
+        List<String> clustersIssues = splitConditions(jsonIssuesConditions,"###clusters");
+
+        String outputJsonString = sanitize(jsonFileToBeSanitized, vmsIssues, hostsIssues,clustersIssues);
+
+        writeStringToFile(outputFileName, outputJsonString);
+
+    }
+
+    private static String readInputFile(String inputFile, String description)
+    {
+        if (inputFile == null)
+        {
+            throw new RuntimeException(description + "path not specified");
+        }
+        try
+        {
+            return new String(Files.readAllBytes(Paths.get(inputFile)), StandardCharsets.UTF_8);
```

Comment:
Could we move `StandardCharsets.UTF_8` to a static final variable? this way we can use the same variable here and also in line 155: `new FileOutputStream(path), "utf-8")))`

Reply:
Rather than store a static final variable from StandardCharsets in a local static final variable, I think it's better to just standardize my usage and use StandardCharsets.UTF_8 in both cases rather than a hard-coded String in one of them.